src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class AbstractFigure implements Figure {
```

Comment:
You should have the following
```suggestion
public abstract class Figure implements Drawable, AreaCalculator
```

Reply:
Не зрозумів взагалі що маю виправити! В мене ж не має Drawable i AreaCalculator

Reply:
Якщо питання не зрозуміле, краще запитай у загальному чаті, тому нам сповіщення не приходять.

Тобі потрібно стоврити Drawable i AreaCalculator і код має виглядати так як на прикладі.